synapse/lib/cell.py

```diff
@@ -494,6 +551,19 @@ async def delBackup(self, name):
         '''
         return await self.cell.delBackup(name)
 
+    @adminapi()
+    async def streamBackupArchive(self, name):
```

Comment:
typically named iterBackupArchive() but i'd be comfortable with simply getBackupArchive() as well.

Reply:
We should also plumb an easy helper for 1-shot take a backup and stream it ( with optional remove it? )

Reply:
Adjusted the naming and added a 1-shot helper